src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/RazorWorkspaceListenerInitializer.cs

```diff
@@ -14,7 +14,7 @@ internal sealed class RazorWorkspaceListenerInitializer
 {
     // This should be moved to the Razor side once things are announced, so defaults are all in one
     // place, in case things ever need to change
-    private const string _projectRazorJsonFileName = "project.razor.vscode.json";
+    private const string _projectRazorJsonFileName = "project.razor.vscode.bin";
```

Comment:
Good idea to update this, but its only a fallback and probably will never be used. The real value to use gets passed in on the command line here: https://github.com/dotnet/vscode-csharp/blob/main/src/razor/src/razorLanguageServerClient.ts#L245

I took the liberty: https://github.com/dotnet/vscode-csharp/pull/6447

Reply:
I was wrong, this is used, but only for writing the file. Reading the file needs the other PR I opened. Sorry, holiday brain.